.github/workflows/MainDistributionPipeline.yml

```
@@ -14,17 +14,19 @@ concurrency:
     name: Build extension binaries
-    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.3.1
+    uses: ./.github/workflows/_extension_distribution.yml
     with:
-      duckdb_version: v1.3.1
-      ci_tools_version: v1.3.1
-      extension_name: fastlanes
\ No newline at end of file
+      duckdb_version: v1.3.2
+      ci_tools_version: v1.3.2
+      extension_name: fastlanes
+      exclude_archs: 'wasm_mvp;wasm_eh;wasm_threads;windows_amd64_rtools;windows_amd64_mingw;linux_amd64_musl;linux_amd64_gcc4;linux_arm64'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)